src/foam/nanos/om/OMDAO.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.nanos.om;
+
+import foam.core.FObject;
+import foam.core.X;
+import foam.dao.DAO;
+import foam.dao.ProxyDAO;
+import foam.mlang.order.Comparator;
+import foam.mlang.predicate.Predicate;
+
+public class OMDAO
```

Comment:
I notice that we don't do anything with selects here. Was that intentional?

Reply:
Yes, noticed the same.  This is copy of PMDAO - which I know explicitly did not include select.